SimplifiedTests/NYPLMyBooksDownloadCenterTests.swift

```diff
@@ -0,0 +1,69 @@
+import XCTest
+
+@testable import SimplyE
+
+class NYPLMyBooksDownloadCenterTests: XCTestCase {
+  func testDeleteBook() {
+    let fileManager = FileManager.default
+    let emptyUrl = URL.init(fileURLWithPath: "")
+    let configs = [
+      [
+        "identifier": "fakeEpub",
+        "type": "application/epub+zip",
+      ],
+      // It looks like audiobooks are handled very differently
+      // [
+      //   "identifier": "fakeAudiobook",
+      //   "type": "application/audiobook+json",
+      // ],
+      [
+        "identifier": "fakePdf",
+        "type": "application/pdf",
+      ]
+    ]
+    for config in configs {
+      let fakeAcquisition = NYPLOPDSAcquisition.init(
+        relation: .generic,
+        type: config["type"]!,
+        hrefURL: emptyUrl,
+        indirectAcquisitions: [NYPLOPDSIndirectAcquisition](),
+        availability: NYPLOPDSAcquisitionAvailabilityUnlimited.init()
+      )
+      let fakeBook = NYPLBook.init(
```

Comment:
Something to consider for the future: in the server code we have a [testing.py file](https://github.com/NYPL-Simplified/server_core/blob/master/testing.py#L306) which has a lot of helper methods for setting up fake data model objects for use in tests.

Right now you're building the iOS test base more or less from scratch, so you don't have anything to build on, but I see the same needs here, and this code looks like it could go into a similar test helper module.

Reply:
I thought about a data setup helper, but I was hesitant to make one until I had more test cases to get an idea of what's needed of such a utility class. But there's no harm in making it anyways and modifying it as we go.

Reply:
I agree it's better to generalize from a few cases rather than one and I think this is fine for now.